base/deprecated.jl

```diff
@@ -243,19 +243,6 @@ DEPRECATED: use @__MODULE__ instead
 end
 export current_module
 
-# PR #22062
```

Comment:
These deprecations were made during the 0.7 cycle so they need to use `@deprecate_moved` or `@deprecate_stdlib` after #25692

Reply:
Hmm, the behavior right now is:

```
julia> Base.LibGit2.prompt("Hello?")
WARNING: Base.LibGit2 is deprecated, run `import LibGit2` instead.
 in module Base
┌ Warning: `LibGit2.prompt(msg::AbstractString; default::AbstractString="", password::Bool=false)` is deprecated, use `result = Base.prompt(msg, default=default, password=password); result === nothing ? "" : result` instead.
│   caller = top-level scope
└ @ Core :0
Hello?:yes
"yes"
```

Isn't that ok?

Reply:
Oh I didn't think that would happen. Neat. You get a deprecation warning for `Base.LibGit2` and then a separate one for `prompt`. So as long as non of these were exported everything should be ok.